go/types/number_util.go

```diff
@@ -4,9 +4,7 @@
 
 package types
 
-import (
-	"math"
-)
+import "math"
```

Comment:
Yeah... `go fmt` fixed them for me. I guess someone forgot to run `go fmt` at some point?